package.json

```diff
@@ -51,5 +51,8 @@
   "engines": {
     "npm": ">=7.0.0",
     "node": ">=14.0.0"
+  },
+  "dependencies": {
+    "bulma": "^0.9.3"
```

Comment:
Each folder will have its own package.json and build. The top level will just be used for development, 

See example repo here https://github.com/vercel/turborepo/tree/main/examples/basic.

Reply:
Fixed in dab8103e9a61b16600a8a8e9d68efbf4fd235ec1